Automattic/wp-calypso
# Add PlanIcon component

Add a generic PlanIcon component. Helps to fix #6759. Replaces the plan icon functionality in `my-sites/plan-features/header`.

Props @gwwar for the SVG icons code. I just copied it from `my-sites/plan-features/header` :).

## Testing Instructions

1. Navigate to `http://calypso.localhost:3000/devdocs/design`;
2. Search for `PlanIcon`;
3. Verify that the plans icons look the same as [here](https://cloud.githubusercontent.com/assets/1464705/16429708/0fa252cc-3d2d-11e6-8141-62a45a98049d.png).

## Screenshots

![selection_056](https://cloud.githubusercontent.com/assets/4988512/16838800/8fcae928-49cb-11e6-9c7f-46c512adf396.jpg)

![selection_057](https://cloud.githubusercontent.com/assets/4988512/16838802/91fa148a-49cb-11e6-844f-ed8523131bc1.jpg)

![selection_058](https://cloud.githubusercontent.com/assets/4988512/16838804/94d6b83e-49cb-11e6-9ece-5a257ed6704d.jpg)

![selection_059](https://cloud.githubusercontent.com/assets/4988512/16838805/97552082-49cb-11e6-8ac0-7a0ffafa7512.jpg)

cc @gwwar 

Test live: https://calypso.live/?branch=add/plan-icon-component
